aibolit/patterns/private_static_method/private_static_method.py

```diff
@@ -0,0 +1,15 @@
+import javalang
+from aibolit.utils.ast import AST
+
+
+class PrivateStaticMethod:
+
+    def __init__(self):
+        pass
+
+    def value(self, filename: str):
+        tree = AST(filename).value().filter(javalang.tree.MethodDeclaration)
+        private_static_methods = list()
+        for node in [node for path, node in tree if all(elem in node.modifiers for elem in ['private', 'static'])]:
```

Comment:
You don't need to iterate over nodes two times. It can be done just once:
```python
private_static_methods = [
    node.position.line
    for _, node in tree
    if all(elem in node.modifiers for elem in ['private', 'static'])
]
```

I also broke down list comprehension to multiple lines to improve readability.

Reply:
I also ignore `path` during iteration since it isn't needed. I done so by assigning yielded value to variable with name `_` (underscore). It is a common convention when you must accept more than you actually need, and want to clarify your intent.